src/measure_theory/integral/bochner.lean

```diff
@@ -1085,6 +1085,22 @@ begin
   simp [ha]
 end
 
+lemma mem_ℒp.snorm_eq_rpow_integral_rpow_norm {f : α → H} {p : ℝ≥0∞} (hp1 : p ≠ 0) (hp2 : p ≠ ∞)
+  (hf : mem_ℒp f p μ) :
+  snorm f p μ = ennreal.of_real ((∫ a, ∥f a∥ ^ p.to_real ∂μ) ^ (p.to_real ⁻¹)) :=
```

Comment:
I am surprised that this proof does not use `measure_theory.snorm_eq_lintegral_rpow_nnnorm`. Was it simply shorter to not use it?
We should make the names of those two lemmas similar by the way. They currently differ by one `rpow`.

Reply:
It doesn't really make a difference to use it or not. I have reformulated the proof to use it. I have also made the names more coherent by removing one `rpow`.